TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Eel/FlowQueryOperations/FilterOperation.php

```diff
@@ -135,6 +135,8 @@ protected function evaluateOperator($value, $operator, $operand)
                 $isOfType = $value->getNodeType()->isOfType($operand[0] === '!' ? substr($operand, 1) : $operand);
                 return $operand[0] === '!' ? $isOfType === false : $isOfType;
             }
+        } elseif ($operator === '!instanceof' && $value instanceof \TYPO3\TYPO3CR\Domain\Model\NodeInterface) {
```

Comment:
Technically no elseif needed, a separate if block would be easier to read, but no blocker, can be refactored any time.